Makefile

```diff
@@ -19,3 +22,22 @@ test-static-build:
 
 test-packaging-tools:
 	@$(MK_DIR)/build_from_docker.sh
+
+$(SNAPCRAFT_FILE): %: %.in Makefile obs-packaging/versions.txt
```

Comment:
I'd like to remove as much as we could the depedency of this file.  Could you use `versions.yaml` from runtime ?

Reply:
mmm but `include` wouldn't work